docs/make.jl

```diff
@@ -2,7 +2,7 @@ using Documenter, Oscar
 
 include("make_work.jl")
 
-Base.invokelatest(BuildDoc.doit, Oscar; warnonly=false, local_build=false, doctest=false)
+Base.invokelatest(BuildDoc.doit, Oscar; warnonly=false, local_build=false, doctest=false, formats = [:html, :pdf], pdf_method = :pdf_via_latex)
```

Comment:
```suggestion
Base.invokelatest(BuildDoc.doit, Oscar; warnonly=false, local_build=false, doctest=false, formats = [:html, :pdf], pdf_method = :pdf_via_docker)
```
Lets try if this works.